### PR TITLE
feat: bump dependencies

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
feat: bump dependencies

**Release note**:
```
NONE
```

### CODE REVIEW

File: modules/create-vm/pkg/vmcreator/vm-creator.go [hunk 1]
```diff
@@ -102,7 +104,16 @@ func (v *VMCreator) createVMVirtctl() (*kubevirtv1.VirtualMachine, error) {
 func runCommand(params string) ([]byte, error) {
 	args := strings.Split(params, " ")
 	output := &bytes.Buffer{}
-	cmd := virtctl.NewCommand()
+	rootCmd := &cobra.Command{
```
[the remark this answers is not in the document]
Comment (0xFelix) [reply modules/create-vm/pkg/vmcreator/vm-creator.go:107]: Can you please be a bit more elaborate and please put it into the commit description?

**Comment by jcanocan** [comment]
Looks like the test suite may have a compile error:
```
 Failed to compile test:
# k8s.io/client-go/applyconfigurations/meta/v1
../vendor/k8s.io/client-go/applyconfigurations/meta/v1/unstructured.go:64:38: cannot use doc (variable of type *"github.com/google/gnostic/openapiv2".Document) as *"github.com/google/gnostic-models/openapiv2".Document value in argument to proto.NewOpenAPIData 
```
Could you please take a look?

**Comment by ksimon1** [comment]
> Looks like the test suite may have a compile error:
> 
> ```
>  Failed to compile test:
> # k8s.io/client-go/applyconfigurations/meta/v1
> ../vendor/k8s.io/client-go/applyconfigurations/meta/v1/unstructured.go:64:38: cannot use doc (variable of type *"github.com/google/gnostic/openapiv2".Document) as *"github.com/google/gnostic-models/openapiv2".Document value in argument to proto.NewOpenAPIData 
> ```
> 
> Could you please take a look?

this is already solved

**Comment by jcanocan** [comment]
> > Looks like the test suite may have a compile error:
> > ```
> >  Failed to compile test:
> > # k8s.io/client-go/applyconfigurations/meta/v1
> > ../vendor/k8s.io/client-go/applyconfigurations/meta/v1/unstructured.go:64:38: cannot use doc (variable of type *"github.com/google/gnostic/openapiv2".Document) as *"github.com/google/gnostic-models/openapiv2".Document value in argument to proto.NewOpenAPIData 
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Could you please take a look?
> 
> this is already solved

Great! Thanks!

**Review by 0xFelix — APPROVED** [review]
/lgtm

Thanks

**Pull request merged**